cmd/kubelet/app/options/globalflags.go

```diff
@@ -73,6 +73,7 @@ func pflagRegister(global, local *pflag.FlagSet, globalName string) {
 }
 
 // registerDeprecated registers the flag with register, and then marks it deprecated
+// nolint:deadcode
```

Comment:
Maybe a better idea to move this function into the _linux.go variant of this file instead of adding the `nolint`

Reply:
Ack. This is done.